plugins/gstperf.c

```diff
@@ -134,11 +148,21 @@ gst_perf_class_init (GstPerfClass * klass)
       g_param_spec_boolean ("print-arm-load", "Print arm load",
           "Print the CPU load info", DEFAULT_PRINT_ARM_LOAD, G_PARAM_WRITABLE));
 
+  g_object_class_install_property (gobject_class, PROP_BITRATE_WINDOW_SIZE,
+      g_param_spec_uint ("bitrate-window-size",
+          "Bitrate moving average window size",
+          "Number of samples used for bitrate moving average window size, 0 is all samples",
+          0, G_MAXINT, DEFAULT_BITRATE_WINDOW_SIZE, G_PARAM_WRITABLE));
+
+  g_object_class_install_property (gobject_class, PROP_BITRATE_FREQUENCY,
+      g_param_spec_uint ("bitrate-frequency",
+          "Bitrate calculation frequency in ms",
```

Comment:
This is misleading, it's not the bitrate calculation frequency, it's the bitrate reporting. The actual bitrate is always computed using bits per second.

Reply:
It's strange having a frequency variable with `ms` units.